docs/devel/DEVELOPMENT.md

```diff
@@ -164,4 +164,5 @@ code is kept in the repository so that commands such as `go get` work.
 If you need to update the version of Go used to build OPA you must update these
 files in the root of this repository:
 
-* `Makefile`- which is used to produce releases locally. Update the `GOVERSION` variable.
+* `.go-version`- which is used by the Makefile and CI tooling. Put the exact go
```

Comment:
Curious why we need a separate file for the GOVERSION. Is this a convention from elsewhere?

Reply:
Not sure if other tools/projects do this, but its a Netlify thing to get it to "gimme" the right version when building the docs. I figure we still have a single spot to define the version.. so meh, its weird, but ok.

Eventually we could migrate away from it if its causing problems.

Reply:
Thanks for clarifying.